### DISCUSSION TITLE
What happens if farming a plot with multiple machines?

### DISCUSSION THREAD

**Discussion by andre-bender** [Chia-Network/chia-blockchain#2456] [Q&A]
Hello,

right now I'm plotting with several machines. Unfortunately the target folder sometimes gets added to farming. I'm wondering what happens when the same plot gets farmed from different machines. Are there any advantages/disadvantages?

**Comment by andre-bender** [comment]
Still no answer? I would like to see some answers because I have multiple harvester and I am not sure if I can add the same plot directories.

**Reply by SackofFluor** [reply]
The plots themselfs are unique, so therefore it shouldnt matter whether the plot is farmed from different machines (at least this is how I understood it).

**Reply by andre-bender** [reply]
Ah okay, so it's actually a pretty great idea to have multiple harvester to get a better redundancy. Cause my Pi crashes sometimes... mostly over night. I've lost a lot of time because of this.

**Reply by nonogrub** [reply]
I have multiple machines running Ubuntu with the same Fingerprint. All final plots go to "one" MAIN Farming Machine.

**Reply by SackofFluor** [reply]
Well, i have to admit i do not know if that really is better or may lead to "hiccups" or any kind of blacklisting? That is something that only specialists may answer :/

I farm with a couple of machines and since I never figured out this "harvester, farmer, full-node" BS mine are all full-node and so far I didn't get a single XCH, so maybe non of my plots is actually working :/

**Reply by andre-bender** [reply]
> I have multiple machines running Ubuntu with the same Fingerprint. All final plots go to "one" MAIN Farming Machine.

What exactly do you mean by that? My Pi is the Main farming machine and I already configured my harvesters with the /ca folder so that they don't run a full node. But I would like to add every plot directory to every harvester. Is that correct?

**Comment by Muffexx** [comment]
As far as I know only one machine should be farming at the same time. 
That is why I disabled UPnP on my secondary PC which is plotting only.
This can be done in the config.yaml file.

Setting up a more elaborate architecture is described [here](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines) but I must admit that that it is not too trivial.

regards
Muffexx
